widget/entry.go

```diff
@@ -39,8 +40,9 @@ var _ fyne.Tabbable = (*Entry)(nil)
 // Entry widget allows simple text to be input when focused.
 type Entry struct {
 	DisableableWidget
-	shortcut fyne.ShortcutHandler
-	Text     string
+	shortcut    fyne.ShortcutHandler
+	Text        string
+	PreeditText string
```

Comment:
Why is this a public field? it feels like it should be an internal implementation if required at all?

Reply:
@andydotxyz OK, privated